### PR TITLE
[AIRFLOW-4035] Remove DagBag from /graph

### PR DESCRIPTION
**Pull request by ffinfo**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title.
  - https://issues.apache.org/jira/browse/AIRFLOW-4035

### Description

Removing DagBag from /graph to make the webserver more stateless

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

File: airflow/www/views.py [hunk 1]
```diff
@@ -87,7 +87,7 @@
     dagbag = models.DagBag
 
 
-def get_date_time_num_runs_dag_runs_form_data(request, session, dag):
+def get_date_time_num_runs_dag_runs_form_data(request, dag):
```
Comment (ashb) [inline airflow/www/views.py:90]: Why the change to use a new session rather than re-use the one that we already have?

**Comment by mik-laj** [comment]
I support this change, but I would like to ask what you think about the introduction of a class-based-view?  This will make the code easier to read and test.

File: tests/www/test_views.py [hunk 2]
```diff
@@ -755,6 +755,7 @@ def setUp(self):
         from airflow.www.views import dagbag
         from airflow.utils.state import State
         dag = DAG(self.DAG_ID, start_date=self.DEFAULT_DATE)
+        dag.sync_to_db()
```
Comment (feng-tao) [inline tests/www/test_views.py:758]: Why do we add many sync_to_db method to different tests?  I saw this kinda change in many of your prs.

Comment (Fokko) [reply tests/www/test_views.py:758]: Good point @feng-tao. Apparently, without the syncing, the tests become flakey. I think we should solve the root cause of this, but this might not be trivial.

Comment (ashb) [reply tests/www/test_views.py:758]: I've done some work on making each test be stand-alone. And without it when running individual tests many tests would fail if the Scheduler tests haven't run, so this sort of thing makes each test less dependent on the other tests, or the order the test are run in.

**Comment by ashb** [comment]
Closing -- handled via Dag Serialization work
